doc: document process for sharing project news (#47189)
nodejs/node: doc/contributing/sharing-project-news.md

# Sharing project news

> The project, supported by the next-10 team, aims to make it easy

Reviewer: ```suggestion
The project aims to make it easy
```

to share project news with those who publish newsletters
that cover Node.js.

> The projects approach is as follows:

Reviewer 2: ```suggestion
The project's approach is as follows:
```

> The project's approach is as follows:
> 

Reviewer: ```suggestion
```

Author: don't think this one makes sense

> * The project will create 1 "What's new in YYYY" issue in the `nodejs\node`

Reviewer 3: ```suggestion
* The project will create one "What's new in YYYY" issue in the `nodejs\node`
```

> * The project will create 1 "What's new in YYYY" issue in the `nodejs\node`

Author: addressed

> * The project will create one "What's new in YYYY" issue in the `nodejs/node`

Reviewer: An issue seems like the wrong tool for the job. Would a GitHub discussion item be better?

> * The project will create one "What's new in YYYY" issue in the `nodejs/node`
>   repo for news related to Node.js core.

Author: ```suggestion
* The project will create one "What's new in YYYY" discussion item in
   the `nodejs/node` repo for news related to Node.js core.
```

>   the `nodejs/node` repo for news related to Node.js core.

Reviewer 4: I think it might be better to use a separate repo for these news. In this way, publications could subscribe to the notifications on that repository to receive the updates.

Reviewer 4: ok!!

* Teams/groups can create a "what's new in YYYY" issue in one or more of their
  repositories. As an example see
  [What's new in the Node-API and node-addon-api world? - 2023 version](https://github.com/nodejs/abi-stable-node/issues/446).
* Collaborators periodically add news as additional comments on these
  issues throughout the year.
* At the end of each year the issue is closed and if appropriate a
  new one is created.
* A list of these documents is maintained the section below as
  quick way to find them.

> * The next-10 team reaches out to teams/groups on an add hoc basis

Reviewer 2: ```suggestion
* The next-10 team reaches out to teams/groups on an ad-hoc basis
```
